### DISCUSSION TITLE
Code adaptation to learn a semi-supervised model with multi-label data

### DISCUSSION THREAD

**Discussion by bundit786** [YGZWQZD/LAMDA-SSL#14] [Q&A]
Hi.
I'm working on a semi-supervised model with multi-label data. I have tried using LAMDA-SSL with my dataset, but it does not seem to support multi-output data in model fitting. I have two questions about the library:
1. Can you guide me on how to modify the code to support multi-label data?
2. In the model training, can we check for what's going on in each loop iteration of labeling, such as the number of labeled samples, the number of iterations and so on (like what we get from sklearn.semi_supervised.self_training)? 

Thank you.
